chpl/chpl-service/src/main/java/gov/healthit/chpl/activity/ActivityMetadataBuilderFactory.java

```diff
@@ -0,0 +1,52 @@
+package gov.healthit.chpl.activity;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.dto.ActivityDTO;
+
+/**
+ * Creates an appropriate metadata buidler object for the type of activity.
+ * @author kekey
+ *
+ */
+@Component
+public class ActivityMetadataBuilderFactory {
+    @Autowired
```

Comment:
Should these be autowired in the constructor?

Reply:
Yes, addressed.